glitter/reminders/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.contenttypes.admin import GenericTabularInline
+from django.utils.html import mark_safe
+
+from .forms import ReminderInlineAdminForm
+from .models import Reminder
+
+
+class ReminderInline(GenericTabularInline):
+    raw_id_fields = ('user',)
+    model = Reminder
+    form = ReminderInlineAdminForm
+    verbose_name_plural = mark_safe('Reminder <small>(to update the content)</small>')
+    extra = 0
+
+    def get_formset(self, request, obj=None, **kwargs):
+        """ Default user to the current version owner. """
+        data = super(ReminderInline, self).get_formset(request, obj, **kwargs)
+        if obj and hasattr(obj, 'current_version'):
```

Comment:
Should this be the owner of the current version, or would it be easier to use `request.user` instead as the initial data, and assume that if you're adding a reminder - you're adding it for yourself?